packages/salesforcedx-vscode-apex/src/messages/i18n.ts

```diff
@@ -56,5 +56,7 @@ export const messages = {
     'Unable to activate the Apex Language Server',
   sobject_refresh_all: 'All SObjects',
   sobject_refresh_custom: 'Custom SObjects',
-  sobject_refresh_standard: 'Standard SObjects'
+  sobject_refresh_standard: 'Standard SObjects',
+  sobjects_refresh_needed: 'To take advantage of autocompletion for SObjects in Apex code, run the SFDX: Refresh SObject Definitions command.',
+  sobjects_refresh_now: 'Run SFDX: Refresh SObject Definitions now'
```

Comment:
FYI @ekapner

Reply:
@jag-sfdc - where and when do these "refresh" messages show up?

Reply:
@ekapner - This pops up in the bottom right part of the editor. I've attached a screen-shot in the "Functionality After" section of the description. This shows up when the user has no sobjects cached locally and has the flag to automatically refresh sobjects turned off.

Reply:
@jag-sfdc - is there a character limit to these messages? Also seems like "sObject" is the proper spelling based on poking around the docs.

If we have space, I recommend:
You don't have any sObjects cached locally. To take advantage of autocompletion for sObjects in Apex code, run SFDX: Refresh sObject Definitions.

For button: remove "now" just -- Run SFDX: Refresh sObject Definitions